build.sbt

```
@@ -84,4 +88,6 @@ lagomKafkaEnabled in ThisBuild := false
 lagomUnmanagedServices in ThisBuild := Map("helloworld.GreeterService" -> s"https://localhost:${`hello-impl-HTTPS-port`}")
 
 
-ThisBuild / javacOptions ++= List("-Xlint:unchecked", "-Xlint:deprecation", "-Werror")
\ No newline at end of file
+def common = Seq(
+  javacOptions in (Compile,compile) ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-parameters", "-Werror")
```

The `compile` isn't necessary.  And the fact that `ThisBuild` doesn't work is yet another instance of https://github.com/sbt/sbt/issues/2899...

```suggestion
  javacOptions in Compile ++= Seq("-Xlint:unchecked", "-Xlint:deprecation", "-parameters", "-Werror")
```